tests/actor/test_battery_status.py

```diff
@@ -713,7 +713,8 @@ async def test_async_battery_status(self, mocker: MockerFixture) -> None:
                 )
                 time.shift(10)
                 await asyncio.sleep(0.3)
-                assert len(status_receiver) == 0
+                with pytest.raises(asyncio.TimeoutError):
+                    await asyncio.wait_for(status_receiver.receive(), timeout=0.1)
```

Comment:
```
Use timeout to check if a receiver is empty

Because checking the length of a receiver appears to be not possible
since channels v1.0.0-rc1.
```

That sounds suspicious, like an existent feature is no longer available and it is not described in the release-notes

Reply:
interesting, it is still there, https://github.com/frequenz-floss/frequenz-channels-python/blob/5cf5b9bbace2524bbe4631921a95b516795b60c1/src/frequenz/channels/_broadcast.py#L421-L427

but I was getting typing errors.  I wonder how it was working until now.  Anyway, will change the commit message.

Reply:
done.

Reply:
Thanks!